craft_providers/lxd/lxc.py

```diff
@@ -1117,8 +1126,9 @@ def check_instance_status(
         # 20 * 3 seconds = 1 minute no change in timer
         timer_queue: deque = deque([-2, -1], maxlen=20)
 
-        # Retry unless the timer queue is all the same
+        # retry until the instance's timer hasn't changed for the last 20 iterations
         while len(set(timer_queue)) > 1:
+            logger.debug("Checking if instance is ready.")
```

Comment:
Not something to fix here, but are these debug logs the reason the cli timer resets every 3 seconds?

Reply:
Correct.  I have that documented here as an enhancement:

https://github.com/canonical/craft-providers/issues/450#issuecomment-1811166186